### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by KamilWasilonek**
https://kamilwasilonek.github.io/ui-internship-lwo2/html/Task_1

### CODE REVIEW

File: html/Task_1/index.html [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+                    <table
+                      width="100%"
+                      cellspacing="0"
+                      cellpadding="5px"
+                      border="1px"
+                    >
+                      <thead>
+                        <tr>
+                          <th>
+                            N<sup><u>o</u></sup>
```
Comment (tsvetkovv) [inline html/Task_1/index.html:102]: wow :)
№

File: html/Task_1/index.html [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+                          <p></p>
+                        </td>
+                      </tr>
+                    </table>
+                  </td>
+                </tr>
+
+                <tr>
+                  <td>
+                    <table>
```
Comment (tsvetkovv) [inline html/Task_1/index.html]: Why do you use table here? 🤔

File: html/Task_1/index.html [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+      href="https://fonts.googleapis.com/css?family=Lato:400,700&display=swap&subset=latin-ext"
+      rel="stylesheet"
+    />
+  </head>
+  <body topmargin="0" leftmargin="0" rightmargin="0">
+    <table width="100%" cellspacing="0" cellpadding="0" border="1px">
+      <!-- Title -->
+      <thead>
+        <th colspan="3" height="98px" align="center">
+          <font size="8">Markup of “Basic HTML” task</font>
```
Comment (tsvetkovv) [inline html/Task_1/index.html]: Better to use `<h*>` elements

File: html/Task_1/index.html [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+                </tr>
+              </tbody>
+            </table>
+          </td>
+
+          <!-- Content -->
+          <td>
+            <table cellpadding="5px">
+              <thead>
+                <th><font size="5">Main Content</font></th>
```
Comment (tsvetkovv) [inline html/Task_1/index.html]: the same, `h*` element

File: html/Task_1/index.html [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+        <th colspan="3" height="98px" align="center">
+          <font size="8">Markup of “Basic HTML” task</font>
+        </th>
+      </thead>
+
+      <tbody>
+        <tr>
+          <!-- Left side -->
+          <td width="198px" valign="top">
+            <table cellpadding="5px">
```
Comment (tsvetkovv) [inline html/Task_1/index.html]: why is table here?

**Review by tsvetkovv — CHANGES_REQUESTED** [review]

**Review by tkachukivan — APPROVED** [review]

**Review by tsvetkovv — APPROVED** [review]
Well done

**Pull request merged**